app/src/modules/insights/routes/panel-configuration.vue

```diff
@@ -19,16 +19,15 @@
 			<v-fancy-select v-model="edits.type" class="select" :items="selectItems" />
 
 			<template v-if="edits.type && selectedPanel">
-				<v-notice v-if="!selectedPanel.options || selectedPanel.options.length === 0">
+				<v-notice v-if="!selectedPanel.options || selectedPanel.options.length === 0 || edits.options === {}">
```

Comment:
This doesn't feel right to me. We wouldn't want to show "No options available" when the user hasn't made any option changes yet? 🤔

Reply:
Yes.